babel.config.js

```diff
@@ -3,7 +3,7 @@ module.exports = function(api) {
 
   return {
     presets: ['@vue/app'],
-    plugins: ['istanbul', '@babel/plugin-proposal-private-methods'],
+    plugins: ['@babel/plugin-proposal-private-methods'],
```

Comment:
It seems that `Istanbul` is installed along with `vue-cli-utils` now. Leaving it here was causing a duplicate plugin warning and causing the unit tests to not run. The only solution was remove it from this array.